Graffle.FlowSdk.Services/GraffleClient.cs

```diff
@@ -128,9 +128,15 @@ public async Task<FlowFullTransaction> GetCompleteTransactionAsync(ByteString tr
                                 TimeSpan.FromMilliseconds(2000)
                             });
 
-            var transactionResult = await retryPolicy.ExecuteAsync(() => GetTransactionResult(transactionId));
             var transaction = await retryPolicy.ExecuteAsync(() => GetTransactionAsync(transactionId));
```

Comment:
@dakota-graffle This line here with the Transaction was the original line mentioned in the card - but the card also mentioned TransactionResult so wasnt 100% sure which one should be set null